.eslintrc.cjs

```diff
@@ -61,7 +61,7 @@ module.exports = defineConfig({
     'node/no-unpublished-require': 'off',
     'node/no-unsupported-features/es-syntax': 'off',
 
-    '@typescript-eslint/ban-ts-comment': 'off', // TODO: we should turn this on in a new PR
+    '@typescript-eslint/ban-ts-comment': 'error',
```

Comment:
```suggestion
    '@typescript-eslint/ban-ts-comment': [
      'error',
      { 'ts-expect-error': 'allow-with-description' },
    ],
```

https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/ban-ts-comment.md#allow-with-description

Reply:
I think the default option has that value.
```ts
const defaultOptions: Options = [
  {
    "ts-expect-error": "allow-with-description",
    "ts-ignore": true,
    "ts-nocheck": true,
    "ts-check": false,
    minimumDescriptionLength: 3,
  },
];
```
https://typescript-eslint.io/rules/ban-ts-comment/#options

Reply:
Oh okay 👌